src/components/chat-view-container/styles.scss

```diff
@@ -3,6 +3,18 @@
 
 // Note: there are some classes in various other places (src/platform-apps/channels/styles.scss)
 
+.chat-view {
+  &__error {
+    padding: 8px;
+    padding-bottom: 32px;
```

Comment:
does this `padding-bottom` get overridden here by the `padding` above?

Reply:
Because it's second the 32 overrides the top one. It's effectively the same as:
```
padding: 8px 8px 32px 8px;
```
but I find it more clear when reading. I'm not so sure if it makes sense though. Thought I'd play with it and see how it felt.